Make "ps" accept (but ignore) all missing common options expected to be supported by "ps"
This is required to not break scripts which are using these options

By Dennis Groenen <tj.groenen@gmail.com> - 2011-08-19
---

--- a/procps/ps.c
+++ b/procps/ps.c
@@ -645,7 +645,7 @@ int ps_main(int argc UNUSED_PARAM, char
 	int w_count = 0;
 	opt_complementary = "-:ww";
 	opts = getopt32(argv, IF_SELINUX("Z")IF_FEATURE_SHOW_THREADS("T")IF_FEATURE_PS_LONG("l")
-					"w", &w_count);
+					"w""ANdeagrxCGUgpstuUojOFfsvuXVmMLScnfyH", &w_count, NULL);
 	/* if w is given once, GNU ps sets the width to 132,
 	 * if w is given more than once, it is "unlimited"
 	 */
@@ -660,7 +660,8 @@ int ps_main(int argc UNUSED_PARAM, char
 # else
 	/* -w is not supported, only -Z and/or -T */
 	opt_complementary = "-";
-	opts = getopt32(argv, IF_SELINUX("Z")IF_FEATURE_SHOW_THREADS("T")IF_FEATURE_PS_LONG("l"));
+	opts = getopt32(argv, IF_SELINUX("Z")IF_FEATURE_SHOW_THREADS("T")IF_FEATURE_PS_LONG("l")
+					"ANdeagrxCGUgpstuUojOFfsvuXVmMLScnfyHw", NULL);
 # endif
 
 # if ENABLE_SELINUX
